panels/class-debug-bar-panel.php

```diff
@@ -60,4 +60,42 @@ function title( $title = null ) {
 	function debug_bar_classes( $classes ) {
 		return $classes;
 	}
+
+	/**
+	 * Render a (floating) block with a title and content.
+	 *
+	 * Generally these blocks are used at the top of a panel for a quick info overview.
+	 *
+	 * {@internal The list of allowed HTML tags is based on the most common HTML tags used
+	 *            by this plugin and its add-ons.}}
+	 *
+	 * @since 0.10.0
+	 *
+	 * @param string $title   The title of the block.
+	 * @param mixed  $content The value/other content for the block.
+	 *
+	 * @return string|void
+	 */
+	protected function render_panel_info_block( $title, $content ) {
+		if ( is_int( $content ) ) {
```

Comment:
This is what would have been the `number_format_i18n()` call.
The reason I dropped it, was that:
* none of the current panels would pass floats
* none of the current panels would lead to large (> 999) numbers

In other words, the `number_format_i18n()` was unnecessary overhead.

It would come back into play if I'd change this to `is_numeric()` which - while not necessary for any of the DB native panels, might make the function more reusable for add-ons, so yes, will change that.